db_json_util.py

```diff
@@ -59,63 +59,76 @@ def main() -> None:
         if args.import_data:
             api_import_from_json_files(path_chains, path_urls, args.url, args.auth_password)
         if args.export_data:
-            if args.json_chains:
-                response = requests.get(args.url + '/all/chains')
+            api_export_to_json_files(path_chains, path_urls, args.url, args.auth_password)
+
+    if args.local:
+        if args.import_data:
+            local_import_from_json_files(path_chains, path_urls, path_db_file)
+        if args.export_data:
+            local_export_to_json_files(path_chains, path_urls, path_db_file)
+
+def api_export_to_json_files(json_chains: Path, json_rpc_urls: Path, api_url: str, auth_pw: str = ""):
+    """
+    Exports data from the SQLite database to JSON files.
+    Assumes the JSON files has a specific format as defined by XYZ.
+    """
+
+    token_reponse = requests.post(api_url + '/token', json={'username': 'dwellir_endpointdb', 'password': f'{auth_pw}'}, timeout=5)
+
+    if token_reponse.status_code != 200:
+        raise requests.exceptions.HTTPError(f'Couldn\'t get access token, {token_reponse.text}')
+    authorization_header = {'Authorization': f'Bearer {token_reponse.json()["access_token"]}'}
+
+    if json_chains:
+                response = requests.get(api_url + '/all/chains', headers=authorization_header, timeout=5)
```

Comment:
Did you run this code successfully? Looks like the indent is one step too far here.

Reply:
Yes, I did. We ran successfully